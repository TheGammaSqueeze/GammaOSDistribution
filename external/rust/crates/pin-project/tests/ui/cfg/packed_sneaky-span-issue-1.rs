use auxiliary_macro::hidden_repr;
use pin_project::pin_project;

#[pin_project]
#[hidden_repr(packed)]
struct S {
    //~^ ERROR may not be used on #[repr(packed)] types
    #[cfg(not(any()))]
    #[pin]
    f: u32,
    #[cfg(any())]
    #[pin]
    f: u8,
}

fn main() {}
