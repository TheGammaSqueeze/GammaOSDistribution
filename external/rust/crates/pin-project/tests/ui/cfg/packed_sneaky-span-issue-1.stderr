error: #[pin_project] attribute may not be used on #[repr(packed)] types
  --> $DIR/packed_sneaky-span-issue-1.rs:6:1
   |
6  | / struct S {
7  | |     //~^ ERROR may not be used on #[repr(packed)] types
8  | |     #[cfg(not(any()))]
9  | |     #[pin]
...  |
13 | |     f: u8,
14 | | }
   | |_^
